### ISSUE TITLE
Why do we need to specify the database path if the manifest contains it?

### ISSUE THREAD

**Issue by hazae41** [orbitdb/orbit-db#647]
Or I didn't understand how it works?

**Comment by phillmac** [comment]
How did you get a manifest?

**Comment by hazae41** [comment]
Here https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md#manifest

**Comment by phillmac** [comment]
In most cases you are given the orbitdb address to begin with, then you have to fetch the manifest afterwards

**Comment by hazae41** [comment]
Yes, so why do we need to add the path in the address if only the hash is needed?

**Comment by RichardLitt** [comment]
Are you talking about the `first-database` section in the `/orbitdb/Qmd8TmZrWASypEp4Er9tgWP4kCNQnW4ncSnvjvyHQ3EVSU/first-database` address?

**Comment by hazae41** [comment]
Yes

**Comment by aphelionz** [comment]
Good question @hazae41 - the database name is required because it's used in the algorithm that generates that very hash. The (very crude) formula for the manifest hash is:

Database Name + ACL + database type.

So it allows you to have multiple databases of the same type, using the same ACL (typically maps to your OrbitDB instance or IPFS node).

However, we should go into more detail about this so I'm going to transfer this issue to the field manual repo for more writing.
